File: PhotoAlbum/app/src/main/java/com/example/photoalbum/MainActivity.java
package com.example.photoalbum;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.RecyclerView;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import com.google.android.material.floatingactionbutton.FloatingActionButton;

public class MainActivity extends AppCompatActivity {

    private FloatingActionButton fab;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        fab = findViewById(R.id.fab);

        fab.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                fab.setOnClickListener(new View.OnClickListener() {
                    @Override
                    public void onClick(View view) {
                        Intent intent = new Intent(MainActivity.this, AddImageActivity.class);
                        startActivity(intent);
                    }
                });
            }
        });
    }
}

File: Messenger/app/src/main/java/com/example/messenger/MainActivity.kt
package com.example.messenger

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val TAG = "MainActivity"

        val userName = userName.text.toString()
        val userEmail = userEmail.text.toString()
        val userPassword = userPassword.text.toString()

        // Register the user
        registerButton.setOnClickListener {

        }

        // Launch login activity
        loginPrompt.setOnClickListener {

        }

    }
}